ui/warlock/inputs.ts

```diff
@@ -55,6 +55,12 @@ export const PetInput = InputHelpers.makeSpecOptionsEnumIconInput<Spec.SpecWarlo
 	changeEmitter: (player: Player<Spec.SpecWarlock>) => player.changeEmitter,
 });
 
+export const NewDPBehaviour = InputHelpers.makeSpecOptionsBooleanInput<Spec.SpecWarlock>({
+	fieldName: 'newDPBehaviour',
+	label: 'New DP behaviour',
```

Comment:
```suggestion
	label: 'Apply new Demonic Pact behavior introduced in 3.4.3',
```

nit: can we elaborate a little more?